ReduxTestTask/Classes/NewsList/ViewModel/NewsListStore.swift

```diff
@@ -67,8 +66,8 @@ extension NewsList {
         case is SelectedCell:
             break
             
-        case let action as ShowArticle:
-            newState.selectedArticle = action.article
+        case is ShowArticle:
```

Comment:
let's remove this if we don't do any work here. We have the default case for this. Same for isSelectedCell